kolibri/core/auth/backends.py

```diff
@@ -71,3 +74,24 @@ def get_user(self, user_id):
             return FacilityUser.objects.get(pk=user_id)
         except FacilityUser.DoesNotExist:
             return None
+
+
+class SessionStore(DBStore):
+    @classmethod
+    def get_model_class(cls):
+        return Session
+
+    def create_model_instance(self, data):
+        obj = super().create_model_instance(data)
+        try:
+            user_id = data.get("_auth_user_id")
+        except (ValueError, TypeError):
+            user_id = None
+        obj.user_id = user_id
+        return obj
+
+    @classmethod
+    def delete_all_sessions(cls, user_ids):
```

Comment:
(nitpick) - If this isn't overriding an existing method, would be clearer to call it `delete_sessions` as `delete_all_sessions` doesn't read like it should take any params but just delete all of the sessions

Reply:
The reason it's called `delete_all_sessions` is because each user can have multiple active sessions, if they have signed in from different devices, so this is to emphasize that this is what is happening.